source/components/Avatar/index.js

```diff
@@ -32,8 +32,8 @@ Avatar.propTypes = {
     /** Badge to display for avatar. */
     badge: PropTypes.oneOf(
         [
-            'admin', 'content-moderator', 'discussion-moderator',
-            'global-discussions-moderator', 'helper', 'staff', 'vstf',
+            'admin', 'content-moderator', 'discussion-moderator','sysop',
```

Comment:
In addition to empty string, `sysop` is also possible. See here: https://github.com/Wikia/pandora/blob/master/service/discussion/src/main/java/com/wikia/discussionservice/services/BadgePermissionsResolver.java#L14